sunpy/net/tests/test_vso.py

```diff
@@ -485,6 +485,15 @@ def test_vso_error(client):
             core_attrs.Instrument('ovsa'))
 
 
+@pytest.mark.remote_data
+def test_vso_post_search(client):
+    results = client.search(a.Time('2017-09-02 23:52', '2017-09-02 23:52:06'), a.Instrument('aia'))
+    res_filtered = results.search(a.Time('2017-09-02 23:52:00', '2017-09-02 23:52:04'))
+    assert len(res_filtered) == 3
+    for rec in res_filtered:
+        assert parse_time(rec.time.start) <= parse_time('2017-09-02 23:52:04')
+
```

Comment:
Can we filter by other attrs other than time?

Reply:
Yes. With instruments , wavelengths too. Adding a test for them.

Reply:
https://github.com/sunpy/sunpy/blob/master/sunpy/net/vso/attrs.py#L327 @nabobalis it will always fail for wavelengths since vso returns a string and not a astropy quantity so `.to(` will always fail. A lot of thing have to be changed about how we handle wavelength in attrs.

The wave attribute looks like this :

```
{
    'wavemin': '6173',
    'wavemax': '6174',
    'waveunit': 'Angstrom',
    'wavetype': 'NARROW'
}
```
I confirmed from VSO api that units can be any of these:

`Angstrom`, `GHz`, `keV`and so making required changes to `attrs.py`.